nav2_util/src/robot_utils.cpp

```diff
@@ -143,6 +143,66 @@ bool getTransform(
   return true;
 }
 
+bool getTransform(
+  const std::string & source_frame_id,
+  const std::string & target_frame_id,
+  const tf2::Duration & transform_tolerance,
+  const std::shared_ptr<tf2_ros::Buffer> tf_buffer,
+  geometry_msgs::msg::TransformStamped & transform_stamped)
```

Comment:
I needed the output as geometry_msgs::msg::TransformStamped to keep the stamp. We can think of a less verbose solution maybe later